src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,8 +56,11 @@ private void configureDefaultCommands() {
      * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
      */
     private void configureButtonBindings() {
-        new LambdaTrigger(() -> OI.leftStick.getTrigger() || OI.rightStick.getTrigger())
+        new LambdaTrigger(() -> OI.leftStick.getTrigger())
         .whileActiveContinuous(new PointInDirectionOfTravel(drive));
```

Comment:
Minor formatting thing, but indent the chain calls in so it's clear they're a continuation of the first line